main/src/library/Int32.flix

```diff
@@ -235,6 +235,16 @@ namespace Int32 {
     ///
     pub def flipBit(x: Int32, position: Int32): Int32 = x ^^^ (1 <<< position)
 
+    ///
+    /// Returns the integer binary logarithm of `x`.
+    ///
+    pub def log2(x: Int32): Int32 = size() - numberOfLeadingZeros(x) - 1
```

Comment:
Cool!

Reply:
I've simplified it further after seeing that `numberOfLeadingZeros` is implemented using `highestOneBitPosition`.